content/cli/smartmodules/hub.md

```diff
@@ -4,6 +4,84 @@ menu: Hub
 weight: 20
 ---
 
-SmartModules Development Kit (SMDK) allows data pipeline operators to build, compile and test SmartModules in their development environment. 
+## Introduction to the SmartModule Hub Service
+
+The SmartModule Hub allows users to start building their data streaming pipline with SmartModules which have been uploaded into the Hub as ready to use packages. The Hub allows Fluvio pipeline construction focusing on the business logic by providing already built, reliable and tested modules. The modules contain high level configurable inputs which can be applied without needing to rebuild the module. Pipelines can employ a mix of Hub downloaded packages as well as custom developed SmartModules.
+
+A view of available and installed SmartModules is accessible from both the fluvio cli as well as on the Fluvio cloud dashboard.
+
+
+## Fluvio Hub Commands
+
+Commands for users to interact with the SmartModule hub are under the **fluvio hub** set of commands.
+
+- `fluvio hub download group/package@version`
+- `fluvio hub list`
+
+### `fluvio hub download`
+
+The command **fluvio hub download <group/package@version>** will download a hub smartmodule into your cluster so it can be configured and used in your pipeline.
+
+```
```

Comment:
Prepend all these with %copy firstline%, a template that will expose "Copy" button and copy first line to the buffer.